tests/test_json.py

```diff
@@ -1,34 +1,39 @@
 # -*- coding: utf-8 -*-
+try:
+    from urllib import urlencode
+    from urllib2 import urlopen, HTTPError
+except ImportError:
+    from urllib.parse import urlencode
+    from urllib.request import urlopen
+    from urllib.error import HTTPError
 
-from urllib import urlencode
-from urllib2 import urlopen, HTTPError
 from json import loads
 
 from logging import log
 
-url = "http://localhost:8001/api/%s"
+url = "http://localhost:8000/api/%s"
 
 class TestJson:
 
     def call(self, name, post=None):
         if not post: post = {}
         post["session"] = self.key
-        u = urlopen(url % name, data=urlencode(post))
+        u = urlopen(url % name, data=urlencode(post).encode('utf-8'))
```

Comment:
Codacy found an issue: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)